test/demo/utils/noteCardUtils.ts

```diff
@@ -0,0 +1,90 @@
+import {ElementHandle} from 'puppeteer';
+import * as moment from 'moment';
+
+import {Notification} from '../../../src/client';
+
+import {promiseMap} from './asyncUtils';
+import {getToastCards} from './toastUtils';
+import {getCenterCardsByNotification} from './notificationCenterUtils';
+
+export interface NotificationCardMetadata {
+    title?: string;
+    body?: string;
+    sourceApp?: string;
+    timeString?: string;
+    icon?: string;
+    buttons?: ButtonMetadata[]
+}
+
+export interface ButtonMetadata {
+    title?: string;
+    iconUrl?: string;
+}
+
+export enum CardType {
+    CENTER = 'center',
+    TOAST = 'toast'
+}
+
+/**
+ * Allow you to get a `Notification`-like object containing the data displayed on a notification card.
+ *
+ * Returns undefined if it cannot find a card for the given uuid/id pair
+ *
+ * Undefined properties imply that the html element for that property could not be found
+ */
+export async function getCardMetadata(card: ElementHandle): Promise<NotificationCardMetadata | undefined> {
```

Comment:
Probably just an oversight on my part. The exported members changed quite a lot over the last few commits as I consolidated/extracted various utils. That being said, there may be a case for keeping this exported as it may be useful to someone's tests down the line? Basically a 🤷‍♂ from me, but could be pushed either way?